### PR TITLE
Rename root->logical domains in TMA doc

### PR DESCRIPTION
**Pull request by zasdfgbnm**

### CODE REVIEW

File: doc/dev/tma/example1.svg
Comment (zasdfgbnm) [inline doc/dev/tma/example1.svg:1]: @rdspring1 How is this file created? I used inkscape to modify it, it was displaying correctly in inkscape, but not rendered correctly in other apps.

Comment (rdspring1) [reply doc/dev/tma/example1.svg:1]: I used powerpoint to create the image.

**Review by rdspring1 — APPROVED** [review]
LGTM

File: doc/dev/tma/example1.svg
Comment (zasdfgbnm) [reply doc/dev/tma/example1.svg:1]: Thanks for updating the file. I have uploaded it.

**Pull request merged**